src/test/java/com/wikia/webdriver/elements/fandom/components/FanFeed.java

```diff
@@ -0,0 +1,22 @@
+package com.wikia.webdriver.elements.fandom.components;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import com.gargoylesoftware.htmlunit.ElementNotFoundException;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+public class FanFeed extends BasePageObject {
+
+  @FindBy(css = "section.fan-feed")
+  private WebElement fanFeed;
+
+  public boolean isDisplayed() {
+    try {
+      return fanFeed.isDisplayed();
+    } catch (ElementNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)